src/framework/audio/internal/soundfontrepository.cpp

```diff
@@ -21,6 +21,17 @@
  */
 #include "soundfontrepository.h"
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <sfloader/fluid_sfont.h>
```

Comment:
Fluid-related code should be in
src/framework/audio/internal/synthesizers/fluidsynth

We should also not mix low-level Fluid logic with high-level logic (questions for the user) in the same class

Reply:
I'm thinking about moving all soundfont related stuff to a new `soundfont` module. That might make some things a bit nicer and more natural. What do you think about that idea?

Reply:
I don't think we need this module, at least for now